GroupDocs.Editor.MVC/Products/Common/Config/ApplicationConfiguration.cs

```diff
@@ -24,23 +21,28 @@ public ApplicationConfiguration()
             YamlParser parser = new YamlParser();
             dynamic configuration = parser.GetConfiguration("application");
             ConfigurationValuesGetter valuesGetter = new ConfigurationValuesGetter(configuration);
-            string license = valuesGetter.GetStringPropertyValue("licensePath");
-            if (String.IsNullOrEmpty(license))
+            string licPath = valuesGetter.GetStringPropertyValue("licensePath");
+            var appRoot = AppContext.BaseDirectory.Substring(0, AppContext.BaseDirectory.LastIndexOf("bin"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=5172828)